src/bikeRental/index.js

```diff
@@ -19,23 +29,20 @@ export function getBikeRentalStation(
 
 export function getBikeRentalStations(
     { host, headers }: HostConfig,
-    distance?: number,
     coordinates: Object,
```

Comment:
Kanskje legge til en egen coordinates flow-type? Man få best utbytte av Flow om man lager type definisjonen, og ikke fallbacker til bruk av Object ;)

Reply:
hehe, jepp! Passer også bra når neste oppgave blir å lage akkurat samme spørring for holdeplasser. Tenker jeg fikser det da 👍❓

Reply:
Den er god! Men da vil jeg se skikkelig pro Flow typer 🍻 😉